docs/dev/controllers/README.md

```diff
@@ -0,0 +1,21 @@
+# Controllers
+
+This document contains the list on controllers in this repository.
+
+Most controllers code can be found under `pkg/controllers` except for some legacy controllers.
+
+| Name                             | Leader             | Description                                           |
+|----------------------------------|--------------------|-------------------------------------------------------|
+| `certmanager-controller`         | :white_check_mark: | Manages TLS certificates                              |
```

Comment:
Should we use :heavy_check_mark: instead of :white_check_mark: , visible easily

Reply:
Done